src/attr_list.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use Attribute;
+use AttrList;
+use ffi;
+use glib::translate::*;
+use std::mem;
+
+impl AttrList {
+    pub fn change(&self, attr: Attribute) {
+        unsafe {
+            ffi::pango_attr_list_change(self.to_glib_none().0, attr.to_glib_none().0 as *mut _);

```

Comment:
That's exactly why I'd like to have https://github.com/gtk-rs/glib/issues/176 :)